indexing-service/src/test/java/org/apache/druid/indexing/common/actions/RetrieveSegmentsActionsTest.java

```diff
@@ -98,31 +102,37 @@ public void testRetrieveUsedSegmentsAction()
   {
     final RetrieveUsedSegmentsAction action =
         new RetrieveUsedSegmentsAction(task.getDataSource(), ImmutableList.of(INTERVAL));
-    final Set<DataSegment> resultSegments = new HashSet<>(action.perform(task, actionTestKit.getTaskActionToolbox()));
-    Assert.assertEquals(expectedUsedSegments, resultSegments);
+    final Set<DataSegment> actualSegments = new HashSet<>(action.perform(task, actionTestKit.getTaskActionToolbox()));
```

Comment:
Suggestion: I typically use nomenclature of `expected` vs `observed`, e.g. `observedUsedSegments`.